### DISCUSSION TITLE
Tradingview does not allow outlook emails to be added for sending alerts

### DISCUSSION THREAD

**Discussion by sahilM1001** [soranoo/TradingView-Free-Webhook-Alerts#63] [General]
Hi @soranoo ,

I have just started using this library to create my alerting feature for trading bots. I have tried to set up a Gmail account as well as an Outlook account, and Tradingview shows that an Outlook account cannot be used for sending alerts as it is not allowed on the Trading View domain.

Is there a way to use Outlook email for receiving emails to the Outlook account? I cannot connect to IMAP Gmail either, as it just shows the user as unauthenticated, even after generating an app password and trying out different things. 

Please help me with this. Looking forward to a quick reply.

**Comment by soranoo** [comment]
Regarding to Outlook,
You may check out the `hotmail.com` domain instead of the `outlook.com`. `hotmail.com` works well for me.

Regarding to Gmail,
I have no issues with it.

Here's my `config.toml`,
![image](https://github.com/soranoo/TradingView-Free-Webhook-Alerts/assets/46896789/7cd26fa2-51c3-441a-ae08-f4e665c121c2)
